README.md

```diff
@@ -29,6 +29,7 @@ In this code pattern, we will be using energy price dataset and analyze the data
 * [Jupyter Notebook](https://jupyter.org/): An open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text.
 
 ## Steps
+1. [Clone the repo](#1-clone-the-repo)
 1. [Create a new project in CP4D](#1-create-a-new-project-in-cp4d)
```

Comment:
your links to steps are off

Reply:
yes. realized after pushing. fixed already. Please review.